src/commands/platforms/create.ts

```diff
@@ -99,37 +129,34 @@ export default class PlatformCreate extends BaseCommand {
 
     const account = await AccountUtils.getAccount(this.app, flags.account, { account_message: 'Select an account to register the platform with' });
 
-    const platform = await this.createArchitectPlatform(flags);
+    const contexts = await this.setupKubeContext(flags);
```

Comment:
`contexts` is a pretty vague variable name especially without a named/specified type. Would prefer `kube_contexts` or something